### PR TITLE
Explain retrieving session statistics

### PR DESCRIPTION
**Pull request by MaChengxin**
Based on the discussion here: https://github.com/arvidn/libtorrent/issues/1953

### CODE REVIEW

File: docs/python_binding.rst [hunk 1]
```diff
@@ -107,6 +107,14 @@ To get a python dictionary of the settings, call ``session::get_settings``.
 
 .. _`library reference`: reference.html
 
+
+Retrieving session statistics in Python is more convenient than that in C++.
+
+In C++, one should call ``post_session_stats()`` in the ``session`` object to get ``session_stats_alert``, in which the statistics is stored as an array.
```
Comment (arvidn) [inline docs/python_binding.rst]: this gives the impression that calling ``post_session_stats()`` is a C++ thing (I think). Would it improve things to move the "in C++" to the next sentence?

File: docs/python_binding.rst [hunk 2]
```diff
@@ -107,6 +107,14 @@ To get a python dictionary of the settings, call ``session::get_settings``.
...
+In order to interpret this array, it is required to call ``session_stats_metrics()`` to get the the indices of these metrics.
+
+In Python it can be done using ``alert.values["NAME_OF_METRICS"]``, where ``alert`` is triggered by ``post_session_stats()``, and ``NAME_OF_METRICS`` is the name of a metric.
```
Comment (arvidn) [inline docs/python_binding.rst]: since you can only look up one counter at a time, perhaps the singular form would be better here, i.e. "NAME_OF_METRIC"

Comment (MaChengxin) [inline docs/python_binding.rst]: Thanks for pointing out!

File: docs/python_binding.rst (hunk 1 above)
Comment (MaChengxin) [inline docs/python_binding.rst]: Yes, that's better.

**Comment by MaChengxin** [comment]
Just made several changes to make the documentation more concise.
Maybe eventually a git squash is needed?

**Pull request merged**